pkg/action/bootentries_test.go

```diff
@@ -664,12 +662,10 @@ var _ = Describe("Bootentries tests", Label("bootentry"), func() {
 				Expect(err).ToNot(HaveOccurred())
 				err = fs.WriteFile("/etc/kairos/branding/grubmenu.cfg", []byte("whatever whatever --id kairos3 {"), os.ModePerm)
 				Expect(err).ToNot(HaveOccurred())
+				Expect(fs.Mkdir("/oem", os.ModePerm)).To(Succeed())
 
 				err = SelectBootEntry(config, "kairos")
 				Expect(err).ToNot(HaveOccurred())
-				Expect(runner.IncludesCmds([][]string{
-					{"grub2-editenv", "/oem/grubenv", "set", "next_entry=kairos"},
-				})).ToNot(HaveOccurred())
```

Comment:
The test wasn't checking much but now it checks even less. Is it possible to check the actual result of setting the default entry using the config.Fs somehow? Some file must have changed, right?

Reply:
done